modules/psap-special-resource-operator-using-CRD.adoc

```diff
@@ -0,0 +1,404 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+:_content-type: PROCEDURE
+[id="deploy-simple-kmod-using-crd"]
+= Building and running the simple-kmod SpecialResource for a hub and spoke topology
+
+You can use the Special Resource Operator (SRO) on a hub and spoke deployment in which {rh-rhacm-first} connects a hub cluster to one or more managed clusters.
+This example procedure shows how the Special Resource Operator builds driver containers in the hub. SRO watches hub cluster resources to identify OCP versions for the helm charts that it uses to create resources which it delivers to spokes.
+
+.Prerequisites
+
+* You have a running {product-title} cluster.
+* You installed the OpenShift CLI (`oc`).
+* You are logged into the OpenShift CLI as a user with `cluster-admin` privileges.
+* You installed the SRO.
+* You installed the Helm CLI (`helm`).
+* You installed {rh-rhacm-first}.
+* You configured a container registry.
+
+.Procedure
+
+. Create a templates directory and change into it.
++
+[source,terminal]
+----
+$ mkdir -p charts/acm-simple-kmod-0.0.1/templates
+----
++
+[source,terminal]
+----
+$ cd charts/acm-simple-kmod-0.0.1/templates
+----
```

Comment:
Commands should not be combined, and also need an intro phrase. I know it seems excessive for simple stuff like this, sorry about that :bow: 

```suggestion
. Create a `templates` directory by running the following command:
+
[source,terminal]
----
$ mkdir -p charts/acm-simple-kmod-0.0.1/templates
----

. Change to the `templates` directory by running the following command:
+
[source,terminal]
----
$ cd charts/acm-simple-kmod-0.0.1/templates
----
```